torch/onnx/symbolic_opset9.py

```diff
@@ -1183,6 +1183,11 @@ def __or_(g, input, other):
     return g.op("Or", input, other)
 
 
+@wrap_logical_op_with_cast_to_and_from("Bool")
+def __xor_(g, input, other):
+    return g.op("Xor", input, other)
```

Comment:
I think we don't export cases for inputs with "non-boolean values". For example:

```
>>> import torch
>>> a = torch.tensor([12])
>>> b = torch.tensor([10])
>>> a ^ b
tensor([6])
```

perhaps should put warning or error out in this case?

Reply:
@BowenBao  Any thing wrong with this example?

Reply:
Yes, all inputs are casted to booleans: using @wrap_logical_op_with_cast_to_and_from("Bool")